docs/using/troubleshooting.md

```diff
@@ -0,0 +1,311 @@
+
+   now accessing the mount should work as expected:
+
+   ```bash
+   # assuming we mounted the volume in /home/jovyan/stagingarea
+   $ ls -ld ~/stagingarea
+   drwxr-xr-x 2 jovyan users 4096 Feb  1 12:55 stagingarea/
+
+   $ touch stagingarea/kale.txt
+   # jovyan is now the owner of /home/jovyan/stagingarea
+   $ ls -la ~/stagingarea/
+   -rw-r--r-- 1 jovyan users    0 Feb  1 14:41 kale.txt
+   ```
+
+   ```{admonition} Additional notes
+      - If you are mounting your volume inside the `/home/` directory, you can use the `-e CHOWN_HOME=yes` and `CHOWN_HOME_OPTS="-R"` flags instead of the `-e CHOWN_EXTRA`
+        and `-e CHOWN_EXTRA_OPTS` in the example above.
+      - This solution should work in most cases where you have created a docker volume (i.e. using the [`docker volume create --name <my-volume>`
+        command](https://docs.docker.com/storage/volumes/#create-and-manage-volumes)) and mounted it using the`-v` flag in `docker run`.
+   ```
+
+2. **Matching the container's UID/GID with the host's**
+
+   Docker handles mounting host directories differently from mounting volumes, even though the syntax is essentially the same (i.e. `-v`).
+
+   When you initialize a Docker container using the flag `-v`, the host directories are bind-mounted directly into the container.
+   Therefore, the permissions and ownership are copied over and will be **the same** as the ones in your local host
+   (including user ids) which may result in permissions errors when trying to access directories or create/modify files inside.
+
+   Suppose your local user has a `UID` and `GID` of `1234`. To fix the UID discrepancies between your local directories and the container's
+   directories, you can run the container with an explicit `NB_UID` and `NB_GID` to match the that of the local user:
+
+   ```bash
+   $ docker run -it --rm \
+       --user root \
+       -p 8888:8888 \
+       -e NB_UID=1234 \
+       -e NB_GID=1234 \
+       -v $(PWD)/test:/home/jovyan/work \
+       jupyter/minimal-notebook:latest
+
+   # you should see an output similar to this
+   Update jovyan's UID:GID to 1234:1234
+   Running as jovyan: bash
+   ```
+
+   where:
+
+   - `NB_IUD` and `NB_GID` should match the local user's UID and GID.
+   - You must use `--user root` to ensure that the `UID` and `GID` are updated at runtime.
+
+   ````{admonition} Additional notes
+   - The caveat with this approach is that since these changes are applied at runtime, you will need to re-run the same command
+     with the appropriate flags and environment variables if you need to recreate the container (i.e. after removing/destroying it).
+   - If you pass a numeric UID, it must be in the range of 0-2147483647
+   - This approach only updates the UID and GID of the **existing `jovyan` user** instead of creating a new user. From the above example:
+     ```bash
+     $ id
+     uid=1234(jovyan) gid=1234(jovyan) groups=1234(jovyan),100(users)
+     ```
+   ````
```

Comment:
```suggestion
   ```{admonition} Additional notes
   - The caveat with this approach is that since these changes are applied at runtime, you will need to re-run the same command
     with the appropriate flags and environment variables if you need to recreate the container (i.e. after removing/destroying it).
   - If you pass a numeric UID, it must be in the range of 0-2147483647
   - This approach only updates the UID and GID of the **existing `jovyan` user** instead of creating a new user. From the above example:
     ```bash
     $ id
     uid=1234(jovyan) gid=1234(jovyan) groups=1234(jovyan),100(users)
     ```
   ```
```